api/src/controllers/roadmap.ts

```diff
@@ -9,24 +9,20 @@ const roadmapsRouter = router({
    * Get a user's roadmap
    */
   get: userProcedure.query(async ({ ctx }) => {
-    const [planners, transfers, timestamp] = await Promise.all([
+    const [planners, timestamp] = await Promise.all([
       db
         .select({ id: planner.id, name: planner.name, content: planner.years })
         .from(planner)
         .where(eq(planner.userId, ctx.session.userId!))
         .orderBy(asc(planner.id)),
-      db
-        .select({ name: transferredMisc.courseName, units: transferredMisc.units })
-        .from(transferredMisc)
-        .where(eq(transferredMisc.userId, ctx.session.userId!)),
```

Comment:
This should also solve the `transferred_misc` duplication issue that we were seeing.